src/ActionCards.lua

```diff
@@ -287,6 +288,36 @@ function ActionCards.get_surpassing_card()
     return surpassing_card
 end
 
+function ActionCards.count_seize_cards()
+    local seize_zone_objects = seize_zone.getObjects()
+    local count = 0
+    for _, obj in ipairs(seize_zone_objects) do
+        if obj.hasTag("Action") and obj.is_face_down then
+            count = count + 1
+        end
+    end
+    return count
+end
+
+function ActionCards.find_seize_player()
+    local seize_zone_objects = seize_zone.getObjects()
+    for _, obj in ipairs(seize_zone_objects) do
+        if obj.hasTag("Action") and obj.is_face_down then
+            local seize_card = ActionCards.get_info(obj)
+            local all_players = Global.getVar("active_players")
+            for _, p in ipairs(all_players) do
+                if p.last_seize_card and
+                   p.last_seize_card.type == seize_card.type and
+                   p.last_seize_card.number == seize_card.number then
+                    return p.color
+                end
+            end
+        end
+    end
+    print("No seize player found despite detecting a seize card")
```

Comment:
Should this be a broadcast? or perhaps use the LOG module?

Reply:
Yes it should be log.warn or similar, I'll give myself a todo to update it.